exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,5 @@
-const name = " Daniel  ";
+const myName = " Daniel  ";
```

Comment:
use "let"

Reply:
Thanks for the feedback, is it for only this one or you mean in general? 
cause if in general then yeah I should of  used Ellie advice to use let :D